core/templates/pages/exploration-player-page/services/exploration-player-state.service.ts

```diff
@@ -75,6 +75,9 @@ export class ExplorationPlayerStateService {
   private _playerStateChangeEventEmitter: EventEmitter<string> = (
     new EventEmitter<string>());
 
+  private _showProgressModalEventEmitter: EventEmitter<boolean> = (
```

Comment:
Avoid starting with a verb for variables. Verbs are generally used in function names because functions perform actions. Can we rename this accordingly? Perhaps take a look at the naming conventions for existing event emitters in the codebase.

Reply:
Does _progressModalShownEventEmitter look fine?

Reply:
Perhaps, _playerProgressModalShownEventEmitter?

Reply:
Done